### PR TITLE
Issue/460editor content loss

### PR DESCRIPTION
**Pull request by maxme**
Fixes https://github.com/wordpress-mobile/WordPress-Editor-Android/issues/460

I also removed some old code (quick pic/video handler) that could lead to post deletion when the user comes back from the media picker.

### CODE REVIEW

**Comment by aforcier** [comment]
The instrumentation tests are currently failing as `EditorFragmentTest` needs to be updated to handle `IllegalEditorStateException` too.

**Comment by aforcier** [comment]
Looks good! Hope this solves the problem.

Left a few comments, should be good to go with those addressed.

**Comment by maxme** [comment]
@aforcier ready for another pass.

File: libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/EditorFragment.java [hunk 1]
```diff
@@ -650,6 +650,7 @@ public void run() {
                         getContent();
                     } catch (IllegalEditorStateException e) {
                         AppLog.e(T.EDITOR, "toggleHtmlMode: unable to get title or content");
+                        toggleButton.setChecked(false);
```
Comment (aforcier) [inline libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/EditorFragment.java]: This needs to be wrapped in a `getActivity().runOnUiThread(...)`.

**Comment by aforcier** [comment]
:shipit:

**Comment by maxme** [comment]
Cherry picked in `release/6.3` (I targeted the wrong branch in this PR)

**Pull request merged**